src/rust/engine/process_execution/src/cache_tests.rs

```diff
@@ -112,18 +112,14 @@ async fn run_roundtrip(script_exit_code: i8) -> RoundtripResults {
 
 #[tokio::test]
 async fn cache_success() {
-  let workunit_store = WorkunitStore::new(false);
-  workunit_store.init_thread_state(None);
-
+  WorkunitStore::setup_for_tests();
```

Comment:
Please feel free to bikeshed this method name, especially if there's something more idiomatic in Rust.